### PR TITLE
Fixing installation at op 1.8.0 and adding new account name in signin…

### PR DESCRIPTION
**Pull request by dtpryce**
… plus device UUID.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,13 +4,17 @@ branches:
     - master
 
 python:
-- '3.7.6'
+- '3.9.2'
```
Comment (PavelZacharuk) [inline .travis.yml:7]: Shouldn't it be 3.7.10? I thought we are using in dockerfiles everywhere `python:3.7.10-stretch`.

Comment (dtpryce) [reply .travis.yml]: Good point ... just added `3.7`, `3.8` and `3.9` which I think selects and runs the highest patch version in travis.

Comment (PavelZacharuk) [reply .travis.yml]: `Downloading the 1Password CLI: op_linux_amd64_v1.8.0.zip`
 ` error: [Errno 13] Permission denied: '/usr/local/bin/op_linux_amd64_v1.8.0.zip28sjoc2z.tmp'`

I don't think it's the python version issue

File: README.md [hunk 2]
```diff
@@ -11,12 +11,23 @@ research and python services use cases.
 pip install 1password
 ```
 
+If you have issues with PyYaml or other disutils installed packages then use:
```
Comment (PavelZacharuk) [inline README.md:14]: is this distutils or disutils?

Comment (dtpryce) [reply README.md:14]: Lol good spelling mistake spot ... ok a quick update

File: .travis.yml (hunk 1 above)
Comment (dtpryce) [reply .travis.yml]: Got that fixed in now in the Travis build ;)

**Review by PavelZacharuk — APPROVED** [review]

**Pull request merged**